vsbot :
# Deep learning parameters
  deeplearning:
    # weights_path: '/home/jc-merlab/Pictures/Data/trained_models/keypointsrcnn_weights_ld_b1_e30.pth'
    weights_path: '/home/jc-merlab/Pictures/Data/trained_models/keypointsrcnn_planning_b1_e50_v8.pth'

# Jacobian Estimation Parameters ------------------------------------
  estimation:
    epsilon: 1.0e-5
    epsilon_a: 1.0e-1 # update threshold or convergence condition
    epsilon_b: 1.0e-5
    gamma:  10.0 # learning rate
    gamma_control_1: [20.0, 10.0, 90.0]
    gamma_control_2: [20.0, 10.0, 90.0] # learning rate during servoing
    # gamma3: 90.0
    # gamma4: 30.0
    mod_err_thresh: [25.0, 25.0]
    window: 50 # window size
    amplitude: 0.1 # amplitude for initial sin velocitiess
    rate: 10 # initialization loop rate

  control:
    saturation: 25
    gain_sm: -3
    thresh1: 25 # visual servoing convergence threshold
    thresh2: 10
    alpha_gamma: 0.0003
    # alpha_gains: 0.02
    # reg_lambda: 0.01
    # jacobian: [19, 47, -79,  7, 89, -13, 71, -59, -23, 61, 41, -67, -3, 2, -11, 29, -43, -31, 73, 17, -5, 83, 37, 53]
    jacobian: [19, 17, -47, 31, -89, 41, 151, -97, -73, 163, 139, 149, -173, 103, 83, -53, 79, 7, -29, 67, 113, -109, 2, 37, 107, -59, 13, 127, 71, 137] #10X3 features
    # jacobian: [19, 47, -79,  7, 89, -13, 71, -59, -23, 61, 41, -67] #4X3 or 6X2  features
    # jacobian: [19, 47, -79,  7, 89, -13, 71, -59, -23, 61, 41, -67,  -3,  2, -11, 29, -43, -31] #6X3 features
    # jacobian: [19, 47, -79,  7, 89, -13] # 2X3 features
   
    gains1: [-3.0,-3.0,-3.0,-3.0,-3.0,-3.0,-3.0,-3.0,-3.0,-3.0] # for intermediate configs
    # gains2: [-3.0,-3.0,-3.0,-3.0,-3.0,-3.0, -7.0, -7.0] # for last config
    gains2: [-3.0,-3.0,-3.0,-3.0,-2.0,-2.0,-2.0,-2.0,-2.0,-2.0]


    debug_mode: 1

# Shape Servoing Parameters -------------------------------------------
  
  shape_control:     
    no_of_features: 10
    no_of_actuators: 3
    num_of_segments: 2